### ISSUE TITLE
Predictable filename for IP Set files

### ISSUE THREAD

**Issue by alecrwaters** [chrissanders/FlowBAT#14]
It'd be handy if IP Set files can have user-defined names (perhaps the same names as the IP Set itself?)

The reason is that sometimes you might want to use multiple IP Sets in an Exclusion and it would be much simpler (and more meaningful) to say "--dipset=/tmp/setOne.rws OR --dipset=/tmp/setTwo.rws" than to work out what the random names are.

**Comment by automayt** [comment]
Agreed. This issue will also serve as a place to fix the "temporary" nature of lists, as they should not be temporary and should not be placed in the user defined temporary directory. In the case that the user chooses /tmp, upstart will clean those files out, and they are no longer accessible until they are regenerated.
